onnxruntime/test/common/tensor_op_test_utils.h

```diff
@@ -1,20 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
 #pragma once
 
-#include<random>
+#include <random>
 
 #include "core/util/math.h"
 #include "test/providers/provider_test_utils.h"
 
 namespace onnxruntime {
 namespace test {
 
-template <class T>
-inline void FillRandom(std::vector<T>& val, T min, T max) {
-  static std::default_random_engine generator;
-  std::uniform_real_distribution<float> distribution(min, max);
-  for (size_t i = 0; i < val.size(); ++i) {
-    val[i] = T(distribution(generator));
+class RandomValueGenerator {
+ public:
+  enum class RandomSeedType {
+    Static,      // static value
```

Comment:
nit: https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
will change to use constants